client/app/telecom/telephony/line/calls/abbreviatedNumbers/telecom-telephony-line-calls-abbreviatedNumbers.controller.js

```diff
@@ -1,77 +1,66 @@
-angular.module("managerApp").controller(
-    "TelecomTelephonyLineCallsAbbreviatedNumbersCtrl",
-    function ($q, $stateParams, $translate, OvhApiTelephony, Toast, PAGINATION_PER_PAGE) {
-        "use strict";
+angular.module("managerApp").controller("TelecomTelephonyLineCallsAbbreviatedNumbersCtrl", class TelecomTelephonyAbbreviatedNumbersCtrl {
```

Comment:
You can refer to my previous comments on `client/app/telecom/telephony/billingAccount/abbreviatedNumbers/telecom-telephony-abbreviatedNumbers.controller.js`